src/MapUpMap.js

```diff
@@ -3,19 +3,40 @@ import Map from 'react-map-gl';
 
 // eslint-disable-next-line import/no-webpack-loader-syntax
 import maplibregl from "!maplibre-gl";
+import axios from "axios";
 
 function MapUpMap() {
+
+  let apiKey = "my-api-key";
+  let serviceUrl = 'https://api.os.uk/maps/vector/v1/vts';
+
+  // BRANDON TEST AXIOS
+  async function find() {
+      axios.get('https://api.os.uk/search/names/v1/find?query=Southampton&key=' + apiKey)
+      .then(function (response) {
+          /* For explanation and debugging purposes we display the full response from the API in the console */
+          console.log(JSON.stringify(response.data, null, 2));
+      });
+  }
+  find()
+  //
+
   return (
     <Map
       initialViewState={{
         latitude: 50.9382,
         longitude: -1.471061,
-        zoom: 16
+        zoom: 2
       }}
       mapLib={maplibregl}
       style={{ width: '100vw', height: '100vh' }}
-      mapStyle="https://basemaps.cartocdn.com/gl/positron-gl-style/style.json"
+      //mapStyle="OS_3857_Light_edited.json"
+      //two other ways to get styles: either through an API call or from github raw. take your pick.
```

Comment:
Hi Brandon - we can't currently do either of these.

API call would mean exposing the API key. We shouldn't really be doing this. If we take the API key out of the code and leave a placeholder then that would break the app (it's currently deployed online).

Essentially we can't use the OS API for vector tiles in this project until we have a means of proxying those calls to protect the API key.